library/src/main/java/com/dslplatform/json/processor/AttributeInfo.java

```diff
@@ -35,6 +35,7 @@
 	public final boolean isGeneric;
 	public final Map<String, Integer> typeVariablesIndex;
 	public final boolean containsStructOwnerType;
+	public TypeMirror genericType;

```

Comment:
Ideally it should be final, unless there is some important reason it cant be

Reply:
Please see my previous comment. I don't know the code well and might be we can remove this parameter.